llvm/test/CodeGen/AMDGPU/divergence-driven-abs.ll

```diff
@@ -15,8 +15,8 @@ define amdgpu_kernel void @s_abs_i32(ptr addrspace(1) %out, i32 %val) nounwind {
 }
 
 ; GCN-LABEL: name: v_abs_i32
-; SI:  V_SUB_CO_U32_e64
-; GFX900: V_SUB_U32_e64
+; SI:  V_SUB_CO_U32_e32
```

Comment:
This is an encoding shrink which is good, but why do none of these changes show a new abs use?